etw/rust/tracelogging_dynamic/src/changelog.rs

```diff
@@ -0,0 +1,29 @@
+//! Release history
+
+#[allow(unused_imports)]
+use crate::*; // For docs
+
+/// # v1.0.3 (2023-03-23, Breaking)

```

Comment:
Because this is a small-but-breaking change to a new product that really can't be said to be v2 yet. Also because the version number would need to propagate to the other crates too, and updating them to v2 with no changes seems weird to me.